netci.groovy

```diff
@@ -124,7 +124,7 @@ def CreateLinuxBuildTasks = { machine, configTag, linuxBranch, nonDefaultTaskSet
 
             def infoScript = 'bash jenkins/get_system_info.sh'
             def debugFlag = buildType == 'debug' ? '--debug' : ''
-            def buildScript = "bash ./build.sh -j=`nproc` ${debugFlag}"
+            def buildScript = "bash ./build.sh -j=`nproc` ${debugFlag} --cxx=/usr/bin/clang++-3.5 --cc=/usr/bin/clang-3.5"
```

Comment:
It should be clang++-3.7 or something newer. 

Actually we may need to test both 3.7 and 3.9

Internally, there are some parts don't comply with the latest updates to clang, however keeping 3.7 is supported is also important.